CCDB/include/CCDB/Backend.h

```diff
@@ -8,7 +8,7 @@
 #include <string>
 #include <vector>
 
-#include "FairMQDevice.h"
+#include <FairMQDevice.h>
```

Comment:
https://gcc.gnu.org/onlinedocs/cpp/Include-Syntax.html

It is just cosmetics here, because the quoted syntax did a fallback to `<>` anyways. You want to use the quoted syntax only for files included from your own source tree.